packages/catalog-search/src/SearchSuggestions.jsx

```diff
@@ -11,17 +13,24 @@ const SearchSuggestions = ({
   handleSuggestionClickSubmit,
   disableSuggestionRedirect,
 }) => {
-  const getLinkToCourse = (course) => `/${enterpriseSlug}/course/${course.key}`;
+  const getLinkToCourse = (course) => {
+    const { learning_type: learningType } = course;
+    if (learningType === LEARNING_TYPE_EXECUTIVE_EDUCATION) {
+      return `/${enterpriseSlug}/executive-education-2u/course/${course.key}`;
```

Comment:
executive-education-2u is a course_type ... could be a constant ... no?

Reply:
Done